src/components/header.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Navbar from './navbar';
+
+const Header = () => (
+  <div className="header">
+    <h1>Math Magicians</h1>
+    <Navbar />
+  </div>
```

Comment:
- [ ] **[OPTIONAL]:** Since this is your Header component, why don't you use the semantic `<header>` tag instead of a regular `<div>`? It would provide a better semantic value for this section of the page. 🤓 🔧 

```suggestion
  <header className="header">
    <h1>Math Magicians</h1>
    <Navbar />
  </header>
```

Reply:
👍